crates/db_schema/src/utils.rs

```diff
@@ -517,6 +528,104 @@ pub fn now() -> AsExprOf<diesel::dsl::now, diesel::sql_types::Timestamptz> {
   diesel::dsl::now.into_sql::<Timestamptz>()
 }
 
+pub type AsRecordOutput<T> = dsl::AsExprOf<T, sql_types::Record<<T as Expression>::SqlType>>;
```

Comment:
Everything below here is pretty hard to read through, so If you can, try to document all these added traits, functions, impls, etc.

Reply:
I documented it and cleaned it up a little

Reply:
Sweet, thx.